src/HotChocolate/Spatial/src/Data/Filters/Convention/GeometryContainsOperationType.cs

```diff
@@ -0,0 +1,20 @@
+using HotChocolate.Data.Filters;
+using HotChocolate.Types;
+using HotChocolate.Types.Spatial;
+using static HotChocolate.Data.Spatial.Filters.SpatialFilterOperations;
+
+namespace HotChocolate.Data.Spatial.Filters
+{
+    public class GeometryContainsOperationType : BooleanOperationFilterInput
+    {
+        protected override void Configure(IFilterInputTypeDescriptor descriptor)
+        {
+            descriptor.Operation(Geometry).Type<NonNullType<GeometryType>>();
+
+            // TODO: Not sure if this makes sense
+            descriptor.Operation(Buffer).Type<FloatType>();
```

Comment:
What is this about?

Reply:
Im not sure if a buffer makes sense on the contains method :) 
i dont have enough domain knowledge for this
@steveoh will know

Reply:
yup, i think it makes sense. If we allow buffering the source, we'll need to think about differentiating between buffering the input vs the source though.